iron-collapse.html

```diff
@@ -142,17 +147,21 @@
     },
 
     updateSize: function(size, animated) {
-      this.enableTransition(animated);
+      this._updateTransition(animated);
       var s = this.style;
       var nochange = s[this.dimension] === size;
       s[this.dimension] = size;
-      if (animated && (nochange || !this._enableTransition)) {
+      if (animated && (nochange || !this.animated)) {
```

Comment:
I'm not sure I understand what checking for both `animated` and `this.animated` does. If the size has changed, then 'transitionEnd' is called never -- `if (animated && !this.animated)` will most of the time be false, no?